docs/authentication-backends.md

```diff
@@ -1,44 +1,40 @@
 # Authentication & Backends
 
-Netlify CMS stores content in your GitHub repository (GitLab and Bitbucket coming soon!). You'll
-need to sign in to your GitHub account through the CMS for this to work, and it requires a server.
-We have a few options for handling this.
+Netlify CMS stores content in your GitHub repository. (GitLab and Bitbucket coming soon!) In order for this to work, you need to authenticate with GitHub, and that requires a server. We have a few options for handling this.
```

Comment:
I liked "You need to sign in to your GitHub account," but it doesn't actually apply to Git Gateway. "Authenticate" is more formal, but also more accurate.